integrations/notion/actions/create-database-row.ts

```diff
@@ -0,0 +1,67 @@
+import type { NangoAction, ProxyConfiguration, RowEntry, SuccessResponse, CreateDatabaseRowInput } from '../../models';
+import { mapPropertiesToNotionFormat } from '../helpers/map-properties.js';
+import { notionPropertySchema } from '../schema.zod.js';
+
+export default async function runAction(nango: NangoAction, input: CreateDatabaseRowInput): Promise<SuccessResponse> {
+    const { databaseId, properties } = input;
+
```

Comment:
We should verify that properties and databaseId are set at the beginning of the action